test/e2e/myReviews.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, expect, Page } from "@playwright/test";
+import { BookPage } from "../pages/BookPage";
+import { MyReviewPage } from "../pages/MyReviewsPage";
+import { LoginPage } from "../pages/LoginPage";
+import { VALID_EMAIL, VALID_PASSWORD } from "../helper-config";
+import { BookDetailPage } from "../pages/BookDetailPage";
+
+let reviewPage: MyReviewPage;
+
+test.beforeEach(async({page}: {page: Page}) => {
+    reviewPage = new MyReviewPage({page});
+    await page.goto(BookPage.MY_BOOKS_URL);
+});
+
+async function performAddReviews(page) {
+    await expect(reviewPage.bookImageLocator).toBeVisible({ timeout: 10000 });
+    const bookDetailPage = await reviewPage.addReviews();
+    await page.goto(BookDetailPage.BOOK_DETAIL_PAGE_URL);
+    await bookDetailPage.reviewIconLink.click();
+    await expect(reviewPage.bookReviewPopupLocator).toBeVisible();
+    await reviewPage.reviewTitleLocator.click();
+    await reviewPage.reviewFeatureLocator.click();
+    await reviewPage.reviewPopupCloseIconLocator.click();
+}
+
+async function performUpdateReviews(page) {
+    await expect(reviewPage.updateReviewsLinkLocator).toBeVisible({ timeout: 10000 });
+    await reviewPage.updateReviews();
+    await expect(reviewPage.bookReviewPopupLocator).toBeVisible();
+    await expect(reviewPage.reviewsContainerLocator).toBeVisible();
+    await reviewPage.selectedReviewCloseIconLocator.click();
+    let selectedReviewsText = await reviewPage.selectedReviewsLocator.allInnerTexts();
+    let cleanedSelectedReviews = selectedReviewsText.map((text)=> text.replace(/×$/, '').trim()); 
+    await reviewPage.reviewPopupCloseIconLocator.click();
+    await page.reload();
+    let listedReviewsText = await reviewPage.listedReviewsLocator.allInnerTexts();
+    expect([...cleanedSelectedReviews].sort()).toEqual([...listedReviewsText].sort());
+}
+
+test('redirects unauthenticated user to login and add/updates review after login', async({page}) => {
+    const loginPage = await reviewPage.goToReviewPage() as LoginPage;
+    await loginPage.navigate();
+    await loginPage.login(VALID_EMAIL, VALID_PASSWORD);
+    await page.goto(MyReviewPage.REVIEW_PAGE_URL);
+    await page.waitForURL(MyReviewPage.REVIEW_PAGE_URL);
+    await page.waitForLoadState('networkidle');
+    await performAddReviews(page);
+    await page.goto(MyReviewPage.REVIEW_PAGE_URL);
+    await page.reload();
+    await page.waitForURL(MyReviewPage.REVIEW_PAGE_URL);
+    await page.waitForLoadState('networkidle');
+    await performUpdateReviews(page);
+});
+
+test("updates review page when user is already logged in", async({page})=> {
```

Comment:
this test also does not assert anything. What should it test and how do we know it worked?